inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -100,4 +113,43 @@
 </div>
 </div>
 
-{% endblock body %}
\ No newline at end of file
+{% endblock body %}
+
+{% block javascript %}
+  {{ super() }}
+  <script type="text/javascript">
+    // require(
+    //   [
+    //     "jquery",
+    //     "js/plots"
+    //   ],
+      
+    //   function(
+    //     $,
+    //     Plots
+    //   ) {
+        
+    //     $(function() { 
+    //       Plots.attachTo(document, context);
+    //     });
+
+    // });
+
```

Comment:
@jmartinm I'm having some issues calling the `js/plots` file above. The error that appears is `Uncaught Error: undefined missing js/plots`.

Using the code inline (see below) works correctly. Adding `"js/plots"` in `inspire_base_init.js` also seems to work correctly.

Any suggestions?